src/test/mochitest/examples/long.js

```diff
@@ -5,70 +5,66 @@ var app = {};
 // may not even be worth separating this logic
 // out, but we do this to demonstrate one way to
 // separate out parts of your application.
-app.TodoModel = function(key) {
+app.TodoModel = function (key) {
   this.key = key;
   this.todos = [];
   this.onChanges = [];
 };
 
-app.TodoModel.prototype.addTodo = function(title) {
-  this.todos = this.todos.concat([
-    {
-      id: Utils.uuid(),
-      title: title,
-      completed: false,
-    },
-  ]);
+app.TodoModel.prototype.addTodo = function (title) {
+  this.todos = this.todos.concat([{
+	id: Utils.uuid(),
```

Comment:
Something's wrong with the indent here and in other parts of this file. It seems like... 5 spaces?